gen/definitions/system.yaml

```diff
@@ -134,7 +134,43 @@ attributes:
       - yang_name: server-ip-list-ordered
         tf_name: servers
         example: 2.3.4.5
-    
+  - yang_name: boot/system/flash/flash-list-ordered-by-user
+    tf_name: boot_system_flash_files
+    type: List
+    exclude_test: true
+    attributes:
+      - yang_name: flash-leaf
+        tf_name: path
+        id: true
+        example: bootflash:c8000v-rpboot.17.15.01a.SPA.pkg
+  - yang_name: boot/system/bootfile/filename-list-ordered-by-user
+    tf_name: boot_system_bootfiles
+    type: List
+    exclude_test: true
+    attributes:
+      - yang_name: filename
+        tf_name: path
+        id: true
+        example: bootflash:c8000v-rpboot.17.15.01a.SPA.pkg
+  - yang_name: enable/secret/secret
+    tf_name: enable_secret
+    no_delete_attributes: true
+    exclude_test: true
+    write_only: true
+    requires_replace: true
+    type: String
+    example: MySecretPassword
+  - yang_name: enable/secret/type
+    tf_name: enable_secret_type
```

Comment:
In this case we don't need to define a `tf_name` here, as by default it would convert the `yang_name` to snake case which would result in the exact same name.